src/main/features/claimant-response/views/court-offer.njk

```diff
@@ -13,12 +13,14 @@
 
       <p>{{ t('The court affordability calculator has determined a new repayment plan as the most the defendant can repay.') }}</p>
 
-      {{
-      repaymentPlanBlock(
+      {% if courtOrderPaymentPlan %}
```

Comment:
this can't be the answer to the problem. This page without a courtOrderPaymentPlan means that the page is meaningless. please find the what is causing the courtOrderPaymentPlan to be empty as it should not be when it gets to this point.

Reply:
That check can probably be removed will test to confirm